Lab6.ipynb

```diff
@@ -0,0 +1,692 @@
+   "metadata": {},
+   "source": [
+    "100,000 Poisson distribution simulation of the LHC background on a regular and a log scaled plot with a true Poisson pmf fitted to it. The fitted curve can be used for later analysis. Both the simulation and pmf have a mean of 100 particle counts."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problem 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to find the 5$\\sigma$ value of the simulated background, I must use the fitted pmf in order to be able to calculate the integral under the curve effectively. The integral corresponds to the probability of measuring a range of values, which is related to the 5$\\sigma$ threshold. Since I used the Poisson distribution (stats.poisson.rvs) to create 100,000 events with a mean count number of 100, the Poisson pmf (stats.poisson.pmf) with a mean of 100 fits the background. I find the number of counts on the Poisson pmf that, when integrated to $\\infty$,  gives a probability associated with measuring a signal 5$\\sigma$ away from the mean or greater of a Gaussian distribution with a mean of 0 and a standard deviation of 1. This gives the upper limit 5$\\sigma$, and to get the lower limit, I do the same thing, just with a probability of -5$\\sigma$ of lower. Since a Gaussian distribution is symmetric, these probabilities are the same. However, since a Poisson distribution is not symmetric except at very large N where the central limit theorem applies, the $\\pm 5\\sigma$ values might not be the same distance away from the mean."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The upper limit 5 sigma theshold is 154.0 and the lower limit 5 sigma threshold is 54.0 for the LHC background.\n"
+     ]
+    }
+   ],
+   "source": [
+    "#using stats.norm.sf to get the probability of measuring 5sigma or greater of a Gaussian\n",
+    "prob_5sigma = stats.norm.sf(5)\n",
+    "\n",
+    "#using stats.poisson.isf and stats.poisson.ppf to get the upper and lower limits, respectively of the 5sigma threshold on the background distribution\n",
+    "upper_threshold = stats.poisson.isf(prob_5sigma, 100)\n",
+    "lower_threshold = stats.poisson.ppf(prob_5sigma, 100)\n",
+    "print('The upper limit 5 sigma theshold is {} and the lower limit 5 sigma threshold is {} for the LHC background.'.format(upper_threshold, lower_threshold))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here it can be seen that the 5$\\sigma$ values are not symmetric about the mean of 100, which is consistent with the characteristics of the Poisson background distribution. They are relatively close to being symmetric, since N=100,000 is a large number of simulations and thus the distribution is approaching that of a Gaussian."
```

Comment:
Same as before, large N of generated Poisson random data does not equal approaching Guassian. It is close to being symmetric because it has a large mean.